controllers/v2/unavailability/api.py

```diff
@@ -53,16 +59,21 @@ def delete(self, user_id, event_id):
 
 class VolunteerUnavailabilityV2(Resource):
 
+    def __init__(self, event_repository):
+        self.event_repository = event_repository
+
     @requires_auth
     @marshal_with(volunteer_unavailability_time)
     @is_user_or_has_role(None, UserType.ROOT_ADMIN)
     def get(self, user_id):
         with session_scope() as session:
-            volunteer_unavailability_record = fetch_event(session, user_id)
+            event_repository = EventRepository(session)
+            volunteer_unavailability_record = event_repository.get_event(user_id)
+            # volunteer_unavailability_record = get_event(session, user_id)
```

Comment:
Please get rid of these commented out lines

Reply:
Yeah, I just deleted them.